Dockerfile

```diff
@@ -49,7 +46,7 @@ COPY . /app
 # Compile sprockets
 RUN if [ "$RAILS_ENV" = "production" ]; then \
     echo "--- :sprockets: Precompiling assets" \
-    && RAILS_ENV=production RAILS_GROUPS=assets bundle exec rake assets:precompile \
+    && RAILS_ENV=production RAILS_GROUPS=assets SECRET_KEY_BASE=xxx bundle exec rake assets:precompile \
```

Comment:
This is a bit weird but it matches up with https://github.com/buildkite/docs/blob/main/docker-compose.production.yml#L11 to get around https://github.com/rails/rails/issues/32947